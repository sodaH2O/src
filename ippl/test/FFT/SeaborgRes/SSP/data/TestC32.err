ATTENTION: 0031-408  32 tasks allocated by LoadLeveler, continuing...
C++ runtime abort: terminate() called by the exception handling mechanism
C++ runtime abort: terminate() called by the exception handling mechanism
C++ runtime abort: terminate() called by the exception handling mechanism
C++ runtime abort: terminate() called by the exception handling mechanism
C++ runtime abort: terminate() called by the exception handling mechanism
C++ runtime abort: terminate() called by the exception handling mechanism
C++ runtime abort: terminate() called by the exception handling mechanism
C++ runtime abort: terminate() called by the exception handling mechanism
C++ runtime abort: terminate() called by the exception handling mechanism
C++ runtime abort: terminate() called by the exception handling mechanism
C++ runtime abort: terminate() called by the exception handling mechanism
C++ runtime abort: terminate() called by the exception handling mechanism
C++ runtime abort: terminate() called by the exception handling mechanism
C++ runtime abort: terminate() called by the exception handling mechanism
C++ runtime abort: terminate() called by the exception handling mechanism
C++ runtime abort: terminate() called by the exception handling mechanism
C++ runtime abort: terminate() called by the exception handling mechanism
C++ runtime abort: terminate() called by the exception handling mechanism
C++ runtime abort: terminate() called by the exception handling mechanism
C++ runtime abort: terminate() called by the exception handling mechanism
C++ runtime abort: terminate() called by the exception handling mechanism
C++ runtime abort: terminate() called by the exception handling mechanism
C++ runtime abort: terminate() called by the exception handling mechanism
C++ runtime abort: terminate() called by the exception handling mechanism
C++ runtime abort: terminate() called by the exception handling mechanism
C++ runtime abort: terminate() called by the exception handling mechanism
C++ runtime abort: terminate() called by the exception handling mechanism
C++ runtime abort: terminate() called by the exception handling mechanism
C++ runtime abort: terminate() called by the exception handling mechanism
C++ runtime abort: terminate() called by the exception handling mechanism
C++ runtime abort: terminate() called by the exception handling mechanism
C++ runtime abort: terminate() called by the exception handling mechanism
