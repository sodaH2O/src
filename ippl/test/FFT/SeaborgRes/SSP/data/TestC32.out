Ippl> CommMPI: Started job 16 on host `s04709'.
Ippl> CommMPI: Started job 17 on host `s04709'.
Ippl> CommMPI: Started job 18 on host `s04709'.
Ippl> CommMPI: Started job 19 on host `s04709'.
Ippl> CommMPI: Started job 20 on host `s04709'.
Ippl> CommMPI: Started job 21 on host `s04709'.
Ippl> CommMPI: Started job 22 on host `s04709'.
Ippl> CommMPI: Started job 23 on host `s04709'.
Ippl> CommMPI: Started job 24 on host `s04709'.
Ippl> CommMPI: Started job 25 on host `s04709'.
Ippl> CommMPI: Started job 26 on host `s04709'.
Ippl> CommMPI: Started job 27 on host `s04709'.
Ippl> CommMPI: Started job 28 on host `s04709'.
Ippl> CommMPI: Started job 29 on host `s04709'.
Ippl> CommMPI: Started job 30 on host `s04709'.
Ippl> CommMPI: Started job 31 on host `s04709'.
Ippl> CommMPI: Parent process waiting for children ...
Ippl> CommMPI: Child 13 ready.
Ippl> CommMPI: Child 11 ready.
Ippl> CommMPI: Child 14 ready.
Ippl> CommMPI: Child 5 ready.
Ippl> CommMPI: Child 3 ready.
Ippl> CommMPI: Child 1 ready.
Ippl> CommMPI: Child 12 ready.
Ippl> CommMPI: Child 2 ready.
Ippl> CommMPI: Child 9 ready.
Ippl> CommMPI: Child 6 ready.
Ippl> CommMPI: Child 7 ready.
Ippl> CommMPI: Child 4 ready.
Ippl> CommMPI: Child 10 ready.
Ippl> CommMPI: Child 8 ready.
Ippl> CommMPI: Child 15 ready.
Ippl> CommMPI: Child 17 ready.
Ippl> CommMPI: Child 18 ready.
Ippl> CommMPI: Child 16 ready.
Ippl> CommMPI: Child 23 ready.
Ippl> CommMPI: Child 29 ready.
Ippl> CommMPI: Child 30 ready.
Ippl> CommMPI: Child 19 ready.
Ippl> CommMPI: Child 27 ready.
Ippl> CommMPI: Child 24 ready.
Ippl> CommMPI: Child 26 ready.
Ippl> CommMPI: Child 28 ready.
Ippl> CommMPI: Child 21 ready.
Ippl> CommMPI: Child 22 ready.
Ippl> CommMPI: Child 25 ready.
Ippl> CommMPI: Child 20 ready.
Ippl> CommMPI: Started job 1 on host `s03909'.
Ippl> CommMPI: Started job 2 on host `s03909'.
Ippl> CommMPI: Started job 3 on host `s03909'.
Ippl> CommMPI: Started job 4 on host `s03909'.
Ippl> CommMPI: Started job 5 on host `s03909'.
Ippl> CommMPI: Started job 6 on host `s03909'.
Ippl> CommMPI: Started job 7 on host `s03909'.
Ippl> CommMPI: Started job 8 on host `s03909'.
Ippl> CommMPI: Started job 9 on host `s03909'.
Ippl> CommMPI: Started job 10 on host `s03909'.
Ippl> CommMPI: Started job 11 on host `s03909'.
Ippl> CommMPI: Started job 12 on host `s03909'.
Ippl> CommMPI: Started job 13 on host `s03909'.
Ippl> CommMPI: Started job 14 on host `s03909'.
Ippl> CommMPI: Started job 15 on host `s03909'.
Ippl> CommMPI: Child 31 ready.
Ippl> CommMPI: Initialization complete.
Configure {0}> Interploation CIC
Configure {0}> All parallel
Dimensionality: D= 3 P= 32 nx= 512 ny= 512 nz= 512
In-place CC transform using all-parallel layout ...
 adding counter 5 event 12 Cycles
 adding counter 0 event 1 Instructions completed
 adding counter 7 event 0 TLB misses
 adding counter 2 event 9 Stores completed
 adding counter 3 event 5 Loads completed
 adding counter 4 event 5 FPU 0 instructions
 adding counter 1 event 35 FPU 1 instructions
 adding counter 6 event 9 FMAs executed


################################################################################
 hpmcount (V 2.4.2) summary (aggregate of 32 POE tasks)
 Average execution time (wall clock time)   : 130.976 seconds
 Average amount of time in user mode        : 83.475000 seconds
 Average amount of time in system mode      : 3.344375 seconds
 Total maximum resident set size            : 6385 Mbytes
 Total shared memory use in text segment    : 310227584 Kbytes*sec
 Total unshared memory use in data segment  : 21687178148 Kbytes*sec
 PM_CYC (Cycles)                            : 996700250505
 PM_INST_CMPL (Instructions completed)      : 759119723061
 PM_TLB_MISS (TLB misses)                   : 11373466        
 PM_ST_CMPL (Stores completed)              : 238798756236
 PM_LD_CMPL (Loads completed)               : 224931759461
 PM_FPU0_CMPL (FPU 0 instructions)          : 36336722124
 PM_FPU1_CMPL (FPU 1 instructions)          : 8021260471
 PM_EXEC_FMA (FMAs executed)                : 6582021078
 Utilization rate                           : 64.076 %
 Avg number of loads per TLB miss           : 19790.9523125
 Load and store operations                  : 463730.514 M
 MIPS                                       : 5856.089
 Instructions per cycle                     : 0.761625
 HW Float points instructions per Cycle     : 0.04440625
 Total Floating point instructions + FMAs   : 50940.005 M
 Total Float point instructions + FMA rate  : 393.033 Mflip/s
 Average FMA percentage                     : 25.84221875 %
 Average computation intensity              : 0.10990625 
################################################################################
